Lndhub check limit

chloehjung15 opened the pull request:
Check transaction limit and daily limit when using lndhub for payment.

PeterRounce commented:
I think this should be reworked with reference to these comments: https://github.com/boltcard/boltcard/issues/51#issuecomment-1486141113

Also fails formatting checks.

chloehjung15 commented:
I've removed checking the daily limit and added `update_payment_paid` function. I've placed the function after the auth and before calling `/payinvoice`. Do you reckon that's the right place for it?

PeterRounce commented:
> #51 (comment)

Yes, `update_payment_paid` looks in the right place.
